src/api/app/controllers/staging/staged_requests_controller.rb

```diff
@@ -31,12 +33,11 @@ def create
   end
 
   def destroy
-    authorize @staging_project, :update?
+    authorize @staging_workflow, :update?
```

Comment:
:thought_balloon: I guess at some point this needs to be more fine grained again and move into `::Staging::StageRequests`

Reply:
true!